corehq/apps/app_manager/static/app_manager/js/forms/app_notifications.js

```diff
@@ -1,14 +1,27 @@
 /* globals hqDefine moment */
 hqDefine('app_manager/js/forms/app_notifications', function () {
-    function NotifyFunction(userId) {
-        return function(msg) {
-            var msgObj = JSON.parse(msg);
-            // only show notifcations from other users
-            if (msgObj.user_id !== userId) {
-                var message = moment(msgObj.timestamp).format('h:mm:ss a') + ': ' + msgObj.text;
-                hqImport("hqwebapp/js/alert_user").alert_user(message, 'info', true);
+    var getMessage = function(redisMessage, userId) {
+        var msgObj = JSON.parse(redisMessage);
+        // only show notifications from other users
```

Comment:
Curious why this was implemented. Does this mean that you could have the form opened in multiple tabs and potentially screw yourself over that way?

no action necessary. I see it was this way before

Reply:
fwiw, it's because every act of opening the form triggers a notification, so without this you'd always see a notification that references yourself and comes from the current tab - but yes, it also means that if you open the form in multiple tabs you're on your own